.husky/pre-commit

```diff
@@ -1,5 +1,2 @@
-#!/usr/bin/env sh
```

Comment:
Decided it would be better to [migrate the syntax to v9](https://github.com/typicode/husky/releases/tag/v9.0.1) in the same pr we update it so we don't lose track of it and have it break on us later if a major update is not backwards compatible with the v8 syntax.